### PR TITLE
Full editing pass

### PR DESCRIPTION
**Pull request by Buzzardo**
I checked spelling, grammar, punctuation, and usage.

Resolves spring-cloud/spring-cloud-dataflow-server-cloudfoundry#385

### CODE REVIEW

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc [hunk 1]
```diff
@@ -2,112 +2,125 @@
 = Tasks on Cloud Foundry
 
 Spring Cloud Data Flow's task functionality exposes new task capabilities within
-the Pivotal Cloud Foundry runtime. It is important to note that the current underlying PCF
+the Pivotal Cloud Foundry runtime.
+
+IMPORTANT: The current underlying PCF
 task capabilities are considered experimental for PCF version versions less than 1.9.  See
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc]: We could remove all the references to PCF 1.9 and the experimental call-outs. It is a very old release and it is not a supported version at Pivotal. Not anymore.

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc [hunk 2]
```diff
@@ -2,112 +2,125 @@
...
+release requires PCF version 1.7.12 or higher to run tasks. Tasks are an experimental
 feature in PCF 1.7 and 1.8 and a GA feature in PCF 1.9.
 
 == Task Database Schema
 
-The database schema for Task applications was changed slighlty from the 1.0.x to 1.1.x version of
-Spring Cloud Task.  Since Spring Cloud Data Flow automatically creates the database schema if it is
-not present upon server startup, you may need to update the schema if you ran a 1.0.x version of the
-Data Flow server and now are upgrading to the 1.1.x version.  You can find the migration scripts
+The database schema for Task applications was changed slightly from the 1.0.x to 1.1.x version of
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc]: The current version is at v1.6. This note is very old as well. We don't want to mention this anymore. The older release ref. guides will include it anyway. If someone asks for it, we can link to the old docs and advise them to upgrade ASAP.

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc [hunk 3]
```diff
@@ -2,112 +2,125 @@
...
 
-Similar to streams, creating a task application is done via the SCDF DSL or through the
-dashboard. To create a task definition in SCDF, you've to either develop a task
+Similar to streams, creating a task application is done by using the SCDF DSL or through the
+dashboard. To create a task definition in SCDF, you must either develop a task
 application or use one of the out-of-the-box link:http://docs.spring.io/spring-cloud-task-app-starters/docs/{sct-starters-core-version}/reference/htmlsingle[task app-starters].
 The maven coordinates of the task application should be registered in SCDF. For more
-details on how to register task applications, review <<_registering_a_task_application,register task applications>>
-section from the core docs.
+details on how to register task applications, see link:https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#spring-cloud-dataflow-register-task-apps[Registering a Task Application]
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/cf-tasks.adoc]: It'd be ideal to not have `current` path in the URL. We have a version as parametrized includes; we can apply the same logic here, so we can be confident the URL won't break.

**Comment by sabbyanandan** [comment]
Looks good! If @cppwfs is happy, it can be merged.

**Comment by cppwfs** [comment]
LGTM

**Comment by cppwfs** [comment]
Rebased, Squashed and merged

**Pull request closed without merging**